src/state/appSessions/index.ts

```diff
@@ -4,6 +4,7 @@ import { Network, ChainId } from '@/chains/types';
 import { createRainbowStore } from '../internal/createRainbowStore';
 
 const chainsIdByNetwork: Record<Network, ChainId> = {
```

Comment:
`chainsIdByNetwork` is only used in an old state migration for `appSession` so we don't need to keep adding chains